api/src/pcapi/core/users/factories.py

```diff
@@ -395,3 +395,14 @@ class EmailValidationEntryFactory(UserEmailHistoryFactory):
 
 class EmailAdminValidationEntryFactory(UserEmailHistoryFactory):
     eventType = users_models.EmailHistoryEventTypeEnum.ADMIN_VALIDATION.value
+
+
+class UserSuspensionFactory(BaseFactory):
+    class Meta:
+        model = users_models.UserSuspension
+
+    user = factory.SubFactory(UserFactory, isActive=False)
+    actorUser = factory.SubFactory(AdminFactory)
+    eventType = users_models.SuspensionEventType.SUSPENDED
+    eventDate = factory.LazyAttribute(lambda o: datetime.utcnow() - relativedelta(days=1))
```

Comment:
(micro-détail) Quand la valeur ne dépend pas de l'objet en construction, on peut utiliser `LazyFunction` au lieu de `LazyAttribute` : 

```suggestion
    eventDate = factory.LazyFunction(lambda: datetime.utcnow() - relativedelta(days=1))
```